features/step_definitions/town_steps.rb

```diff
@@ -56,3 +56,8 @@
 Then(/^Git Town is (?:no longer|still not) configured for this repository$/) do
   expect(git_town_configuration).to be_empty
 end
+
+
+Then(/^I see the first question of the configuration wizard$/) do
+  step %Q(I see "Please specify the main dev branch")
```

Comment:
Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.